pkg/sanity/controller.go

```diff
@@ -966,8 +908,9 @@ var _ = DescribeSanity("Controller Service [Controller Server]", func(sc *TestCo
 			_, err = c.ValidateVolumeCapabilities(
 				context.Background(),
 				&csi.ValidateVolumeCapabilitiesRequest{
-					VolumeId: vol.GetVolume().GetVolumeId(),
-					Secrets:  sc.Secrets.ControllerValidateVolumeCapabilitiesSecret,
+					VolumeId:           vol.GetVolume().GetVolumeId(),
+					VolumeCapabilities: nil,
```

Comment:
Is this necessary?

Reply:
No, it is not.
But the test case name is `should fail when no volume capabilities are provided` that is why I decided to emphasize that `VolumeCapabilities` is nil.